### ISSUE TITLE
X-plosion Tankz™

### ISSUE THREAD

**Issue by ethanguin** [nathan/phosphorus#559]
It doesn't work on any of the computers I try to use it on.

http://scratch.mit.edu/projects/34812460/
http://phosphorus.github.io/#34812460
Mozilla/5.0 (Windows NT 5.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

**Comment by htmlgames** [comment]
Uncaught Error: Dynamic variables are not supported

Does this relate to using hacked blocks in Scratch?

I can bypass this error using the debugger, but the game shows but cannot be played. Also complains about hide / show list missing.

**Comment by ethanguin** [comment]
Oh, that makes perfect sense, I actually did use the <if (variable) key
pressed> block. So that would probably make the error.
Thanks,
~Ethanguin

*~ETHANGUIN*

**Comment by tjvr** [comment]
> Dynamic variables are not supported

@nathan Are we planning to support dynamic variables? If not, I suggest closing this!

**Comment by nathan** [comment]
Yeah, there's a way to compile them so that dynamic lookups take a slow route and static lookups are just as fast. I just haven't gotten around to actually implementing the dynamic part yet.
